src/test/java/domain/CarTest.java

```diff
@@ -0,0 +1,62 @@
+package domain;
```

Comment:
테스트 클래스는 대응되는 프러덕션 클래스와 동위에 위치해야해요. 

```suggestion
package racingcar.domain;
```

Reply:
페어 프로그래밍 중 프로젝트 구조가 변경 되면서 발생한 실수인 것 같습니다. 앞으로도 명시하겠습니다!!

Reply:
좋습니다~~! 프로젝트 구조가 복잡하다보면 저도 종종 실수할 때가 있어요. 신경써보도록하죠~~!